### PR TITLE
Fixed bug where registering would break seeder

### PR DESCRIPTION
**Pull request by SamAlrahhal**

### CODE REVIEW

File: resources/views/profile/edit.blade.php [hunk 1]
```diff
@@ -19,12 +19,6 @@ class="text-xl font-semibold leading-tight text-gray-800 dark:text-gray-200">
           @include('profile.partials.update-password-form')
         </div>
       </div>
-
-      <div class="bg-white p-4 shadow dark:bg-gray-800 sm:rounded-lg sm:p-8">
-        <div class="max-w-xl">
-          @include('profile.partials.delete-user-form')
-        </div>
-      </div>
```
Comment (ThomasFrans) [inline resources/views/profile/edit.blade.php:27]: Why does this have to go? There is soft deletion for users, so the database won't be corrupted. Shouldn't the users have the ability to remove their account? Maybe there needs to be a validation to make sure they want to remove it, but for now can't this stay?

**Review by ThomasFrans — COMMENTED** [review]
The name change stuff seems to work now. Just one comment on the other part.

File: resources/views/profile/edit.blade.php (hunk 1 above)
Comment (SamAlrahhal) [reply resources/views/profile/edit.blade.php:27]: The teacher told me that the users shouldn't be able to delete themself. But your suggestion is even better.

Comment (ThomasFrans) [reply resources/views/profile/edit.blade.php]: I don't want to argue against what Phaedra says, but it just seems weird to lock users on the website with no way to delete an account. GDPR would not be happy with that :smile:

**Review by ThomasFrans — APPROVED** [review]

**Pull request merged**